### PR TITLE
done with hometask 2

### PR DESCRIPTION
**Pull request by YanaLapitskaya**

### CODE REVIEW

File: task_2_2/main.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package main
+
+import "fmt"
+
+// Write max([]string) function that returns the longest word from the slice of strings (the first if there are no more than one)
```
Comment (VolhaVetrava) [inline task_2_2/main.go:5]: Best practice is to start comments with the name of the method, then what it does
https://blog.golang.org/godoc

**Review by VolhaVetrava — APPROVED** [review]

File: task_2_2/main.go (hunk 1 above)
Comment (YanaLapitskaya) [reply task_2_2/main.go:5]: oki, thank you

Comment (YanaLapitskaya) [reply task_2_2/main.go:5]: oki, thank you, will take into account next time

Comment (YanaLapitskaya) [reply task_2_2/main.go:5]: oki, thank you

**Review by ikamornikau — APPROVED** [review]

**Pull request merged**